plugins/inputs/fireboard/README.md

```diff
@@ -0,0 +1,56 @@
+# Fireboard Input Plugin
+
+The fireboard plugin gathers the real time temperature data from fireboard
+thermometers.  In order to use this input plugin, you'll need to sign up
+to use their REST API, you can find more information on their website
+here [https://docs.fireboard.io/reference/restapi.html]
+
+### Configuration
+
+This section contains the default TOML to configure the plugin.  You can
+generate it using `telegraf --usage <plugin-name>`.
+
+```toml
+[[inputs.fireboard]]
+  ## Specify auth token for your account
+  ## https://docs.fireboard.io/reference/restapi.html#Authentication
+  # authToken = "b4bb6e6a7b6231acb9f71b304edb2274693d8849"
+  ## You can override the fireboard server URL if necessary
+  # URL = https://fireboard.io/api/v1/devices.json
```

Comment:
Use the `snake_case` version of the options, it would be best to add struct tags to the enforce this naming, I'll add a note on the struct.